mitmproxy/addons/maplocal.py

```diff
@@ -1,52 +1,86 @@
 import mimetypes
 import re
 import typing
-import urllib
 from pathlib import Path
+
 from werkzeug.security import safe_join
 
-from mitmproxy import ctx, exceptions, http
-from mitmproxy.addons.modifyheaders import parse_modify_spec, ModifySpec
+from mitmproxy import ctx, exceptions, flowfilter, http
+from mitmproxy.addons.modifyheaders import parse_spec
+
+
+class MapLocalSpec(typing.NamedTuple):
```

Comment:
Having separate specs is a good idea, it got a bit messy trying to use the `ModifySpec` for the `Map*` addons. :+1: